views/app/adminusers.ejs

```diff
@@ -63,9 +63,7 @@
                    }
                 }
             };
-            var urlParams = new URLSearchParams(window.location.search);
-            var securityRating = urlParams.get('securityRating') 
-            xmlhttp.open("GET", "/app/admin/usersapi/" + securityRating, true);
```

Comment:
Eww. Actually this was the whole point of the refactor. Isn't it much better without having to query the current URL params in a view??